cypress/support/generate.js

```diff
@@ -0,0 +1,12 @@
+import { faker } from '@faker-js/faker';
+
+function generateUser() {
+  const randomNumber = Math.random().toString().slice(2, 6);
+  const username = faker.internet.userName() + '_' + randomNumber;
+  const email = `${username}@mail.com`;
```

Comment:
You're generating a random number and appending it to the username and email. This is a good practice to ensure unique values. However, it might be a good idea to also append this random number to the password to ensure unique and hard-to-guess passwords.